tiny-example/model_description.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Model
```

Comment:
Just realised that this file should be called `model_definition.yaml`

Reply:
Nice catch.

Reply:
Although, I did think we could call these anything. I was going to add on to the OpenUDM file called dafni_model_definition.yaml (Just so it's obvious where it is to be used)

Reply:
You're right, these can be called anything you want them to be. I just think it would be good to name them consistently and clearly with the other terminology we are trying to teach the users. It is harder to follow if in the guides we are refering to Model Definition files when the file is called actually called `model_description`